symfony/psr-http-message-bridge/2.2/config/packages/psr_http_message_bridge.yaml

```diff
@@ -0,0 +1,5 @@
+psr_http_message:
+
+    # Message converters that allow controllers to operate on PSR-7 requests/responses
+    message_converters:
+        enabled: false
```

Comment:
Do we need any config file? In practice, of a library depends on `symfony/psr-http-message-bridge`... they would also get the bundle, which would give them this config file. New config files cause bloat. I'd prefer to not ship any config file. It already looks like it's usually not needed. I also just suggested the possibility of even enabling this part by default (https://github.com/symfony/psr-http-message-bridge/pull/94/files#r592669310) but I don't know if there are downsides to making this the default :)

Reply:
Enabling by default means registering an event listener on the hotpath, better not.
But I share the feeling about the recipe, it might not be needed.

Reply:
Ok, I'd vote for disabling the listeners by default (as they are done already) but no config file in the recipe.